src/installer.ts

```diff
@@ -373,6 +373,10 @@ async function queryDistForMatch(
   let versions: string[] = [];
   let nodeVersions = await getVersionsFromDist();
 
+  if (versionSpec === 'node') {
```

Comment:
I'd rather go with `latest` here.  I get that there's prior art for `node`, but it doesn't make much sense to me on its own merits.

The meaning of `latest` is self-evident and matches other usage in Actions (`ubuntu-latest`, etc.)

Reply:
[NVM](https://github.com/nvm-sh/nvm) does not support `latest`.
So if you use a `.nvmrc` file, `latest` is not an option.

From the [nvm docs](https://github.com/nvm-sh/nvm#usage):

> …
> 
> - [node](https://nodejs.org/en/): this installs the latest version of node
> - iojs: this installs the latest version of [io.js](https://iojs.org/en/)
> - stable: this alias is deprecated, and only truly applies to node v0.12 and earlier. Currently, this is an alias for node.
> - unstable: this alias points to node v0.11 - the last "unstable" node release, since post-1.0, all node versions are stable. (in SemVer, versions communicate breakage, not stability).
> 
> …

I would prefer to use the same aliases as NVM.